packages/libs/react-ui/src/components/Icons/IconWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import { sizeVariants } from './IconWrapper.css';
+
+import React, { SVGProps } from 'react';
+
+export interface IIconProps {
+  size?: keyof typeof sizeVariants;
+}
+
+export type IconType = SVGProps<SVGSVGElement> & IIconProps;
+
+export const IconWrapper = (
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  Component: React.FC<SVGProps<SVGSVGElement>>,
+): React.FC<IconType> => {
+  const WrappedIcon: React.FC<SVGProps<SVGSVGElement> & IIconProps> = ({
+    size = 'md',
+    ...props
+  }) => (
+    <div className={sizeVariants[size]}>
```

Comment:
this used to be a span.
And it looks like it is still a display:'block'

Reply:
good catch